NeuroID/src/main/java/com/neuroid/tracker/NeuroID.kt

```diff
@@ -58,6 +58,7 @@ class NeuroID
     private constructor(
         internal var application: Application?,
         internal var clientKey: String,
+        internal var isRN: Boolean,
```

Comment:
Why is this isRN? I thought we were setting the ADV `isAdvancedDevice` variable? isRN is set in a different method from the react-native sdk

Reply:
also should these be `val` instead of `var ` because we don't want to change clientKey or `isAdvancedDevice` after its set?

Reply:
fixed. sorry bout that.